actionpack/lib/action_controller/metal/implicit_render.rb

```diff
@@ -8,6 +8,8 @@ def send_action(method, *args)
 
     def default_render(*args)
       render(*args)
+    rescue ActionView::MissingTemplate
```

Comment:
Concerns around this being slow / an exceptional state when it doesn't need to be. It should be once-per-request code path, and it was exceptional but being handled. 

I'm happy to check if the template exists or not, just need to find out how to turn a method into the correct template name.